client/src/App.tsx

```diff
@@ -32,11 +32,11 @@ function App(): JSX.Element {
                 <Route exact path="/login" component={Login} />
                 <Route exact path="/signup" component={Signup} />
                 <Route exact path="/editprofile" component={EditProfile}></Route>
```

Comment:
You can make this a self closing route :)

Reply:
Whoops, nice catch again!